csi/server/plugin/opensds/fileshare.go

```diff
@@ -197,7 +197,8 @@ func (f *FileShare) ControllerPublishFileShare(req *csi.ControllerPublishVolumeR
 		attachMode = "read,write"
 	}
 
-	accessTo := strings.Split(req.GetNodeId(), ",")[IpIdx]
+	nodeInfo := strings.Split(req.GetNodeId(), ",")
```

Comment:
One suggestion, and it's generic, When we have any code for parsing o/p, IMHO, it should be better to put the command/API o/p in the comment (replacing non-shareable text with place holder). Helps